README.md

```diff
@@ -21,7 +21,7 @@
 ## Installing the UI5 CLI
 ### Requirements
 - [Node.js](https://nodejs.org/) (**version must be 8.3 or higher** ⚠️)
-- Optional: Yarn ([yarnpkg.com](https://yarnpkg.com/en/docs/install))  
+- Optional: [Yarn](https://yarnpkg.com/en/docs/install) (**version must be 1.0 or higher**)  
```

Comment:
Why yarn is optional? Until ui5 packages are published to npm registry it's more than required to make it all work, isn't it?

Reply:
True. Right now it's required - and that's described in the line below.